build.py

```diff
@@ -109,16 +109,20 @@ def create_package_fs(build_root):
         create_dir(os.path.join(build_root, d))
         os.chmod(os.path.join(build_root, d), 0755)
 
-def package_scripts(build_root):
+def package_scripts(build_root, static=False):
```

Comment:
> Do you think all tars and zips should be this way?

No, I think it is useful to provide the scripts in tars, zips since they have value, at least in the linux versions. To me the contents of a package (tar, deb, zip etc)  depends on the platform and not the package type. No matter the package type provide the relevant content for the platform. 

Why do Windows users want static builds? I get why they don't care about service scripts etc but not static builds.